enos/provider/g5k.py

```diff
@@ -165,37 +166,87 @@ def _make_reservation(self):
         criteria = {}
         # Actual criteria are :
         # - Number of node per site
+        queues = self._get_queues()
```

Comment:
But what about complexity??
In the case where we have two queues, that's four iterations instead of two!! That is twice as many iterations!!